chart/templates/mayastor/apis/api-rest-deployment.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     app: api-rest
 spec:
-  replicas: 1
+  replicas: {{ .Values.apis.rest.replicaCount }}
```

Comment:
Can we have the earlyEvictionToleration and PriorityClass for rest as well?

Reply:
Since we can run multiple instances of rest I have not added it but we can add for that as well.

Reply:
Done.